src/azure-cli/azure/cli/command_modules/resource/_json_handler.py

```diff
@@ -0,0 +1,220 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def json_min(js, **kwargs):
+    """
+    returns a minified version of the json string
+    """
+    import io
+    klass = io.StringIO
+    ins = klass(js)
+    outs = klass()
+    JsonMinify(ins, outs, **kwargs).minify()
+    return outs.getvalue()
+
+
+# pylint: disable=attribute-defined-outside-init, too-many-branches, too-many-statements, too-many-instance-attributes
```

Comment:
We can disable all checks with

```
# pylint: skip-file
```

Reply:
I'd like to put the JSON related functions in this file in the future, so I just disable the `pylint` of this function for the time being